docs.kosli.com/content/faq/_index.md

```diff
@@ -242,3 +242,17 @@ kosli attest generic Dockerfile true ...
 ```
 The parser then sees `Dockerfile` and `true` as the two
 arguments to `kosli attest generic`.
+
+## Fingerprint calculation
+
+In order to calculate the fingerprint for an artifact, Kosli requires the path to the relevant file or directory, or the relevant image name. The command typically takes the form:
+```
+kosli attest generic [IMAGE-NAME | FILE-PATH | DIR-PATH] [flags]
+```
+
+When using multi-line commands in the shell or a script, if the image name or path has not been provided as above and whitespace has unintentionally been added after the backslash on one of the lines, this whitespace is interpreted as the name or path.
```

Comment:
...after the line-continuation backslash...
...this whitespace character is interpreted as...